connect-base/1.6/docker-entrypoint.sh

```diff
@@ -125,6 +125,38 @@ if [[ -n "$JMXPORT" && -n "$JMXHOST" ]]; then
     export KAFKA_JMX_OPTS="-Djava.rmi.server.hostname=${JMXHOST} -Dcom.sun.management.jmxremote.rmi.port=${JMXPORT} -Dcom.sun.management.jmxremote.port=${JMXPORT} -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.authenticate=${JMXAUTH} -Dcom.sun.management.jmxremote.ssl=${JMXSSL} "
 fi
 
+#
+# Setup Flight Recorder
+#
+if [[ -n "$JFR_RUN_FILENAME" ]]; then
```

Comment:
@jpechane, filename isn't a mandatory option. You can start a continuous recording (which you then can dump using JMC for instance) solely via -XX:StartFlightRecording. I.e. we cannot rely on presence of JFR_RUN_FILENAME. How about this:

* Add an env var `ENABLE_JFR`, akin to `ENABLE_DEBEZIUM_SCRIPTING` and `ENABLE_APICURIO_CONVERTERS`
* If true (the default), the `-XX:StartFlightRecording` will be passed
* any variables prefixed with `JFR_RECORDING` (I think that's better than "RUN") or `JFR_OPT` will be passed through to `-XX:StartFlightRecording` and `-XX:FlightRecorderOptions`, respectively
* If there's at least one `JFR_OPT` variable, `-XX:FlightRecorderOptions` will be set (i.e. that's independent of `-XX:StartFlightRecording`)

That way, there'll be a continuous recording enabled by default, using default settings (e.g. max buffer size of 250 MB). WDYT?

Reply:
@gunnarmorling Done